css.md

```diff
@@ -52,6 +53,18 @@ This style guide focusses primarily on the usage of SASS, which is the standard
     :font-size 0.3em
   ```
 
+* Top-level numeric calculations should always be wrapped in parentheses
+
+  ```sass
+  // good
+  .foo
+    width: (100% / 3)
+
+  // bad
+  .foo
+    width: 100% / 3
+  ```
```

Comment:
The parentheses tell SASS that it is a math calculation and not an actual value. It's required in many instances to make the compiler actually perform a calculation.

## Input
```sass
.example-1
  font: 12px/1.5 sans-serif // 12px is font size, 1.5 is line-height
  width: 12px/1.5

.example-2
  font: (12px/1.5) sans-serif
  width: (12px/1.5)
```

## Output
```css
.example-1 {
  font: 12px/1.5 sans-serif;
  width: 12px/1.5;
}

.example-2 {
  font: 8px sans-serif;
  width: 8px;
}
```